### DISCUSSION TITLE
React npm package mui-modal-provider like Hooks for Mantine Modal or Dialog

### DISCUSSION THREAD

**Discussion by amitdvs** [mantinedev/mantine#1741] [Feature requests]
Requesting a new mantine hook to use modal or dialog by code

There is an npm package mui-modal-provider for MUI that help to easy way to open and close the modal. The package is using context and hooks to open and close.

https://www.npmjs.com/package/mui-modal-provider

**Comment by zhulien-ivanov** [comment]
Take a look at `@mantine/modals`: https://mantine.dev/others/modals/.

**Reply by amitdvs** [reply]
Thanks @zhulien-ivanov
